expo/tests/gbe/test_bid_costume.py

```diff
@@ -61,37 +88,46 @@ def test_costume_bid_post_form_not_valid(self):
         login_as(PersonaFactory().contact, self)
         data = self.get_costume_form(invalid=True)
         response = self.client.post(url)
-        nt.assert_equal(response.status_code, 200)
-        nt.assert_true('Displaying a Costume' in response.content)
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue('Displaying a Costume' in response.content)
 
     def test_costume_bid_post_with_submit(self):
-        '''costume_bid, not submitting and no other problems,
+        '''costume_bid, submitting and no other problems,
         should redirect to home'''
-        url = reverse(self.view_name,
-                      urlconf="gbe.urls")
-        login_as(PersonaFactory().contact, self)
-        data = self.get_costume_form(submit=True)
-        response = self.client.post(url, data=data, follow=True)
-        nt.assert_equal(response.status_code, 200)
-        nt.assert_true(('http://testserver/gbe', 302)
-                       in response.redirect_chain)
-        nt.assert_true("Your Account" in response.content)
+        response, data = self.post_costume_submission()
+        self.assertEqual(response.status_code, 200)
```

Comment:
When I read:

https://github.com/bethlakshmi/GBE2/pull/789

My takeaway was that we should use standard python asserts.  I assumed that asserts coming from the django.test.TestCase would be standard enough and that it would be worthwhile to clear out the dependancy on nose.

Fine.  Will stop it in the future - do you want me to pull that part of these changes?